### DISCUSSION TITLE
Han el all preset

### DISCUSSION THREAD

**Discussion by alexell22** [milaboratory/mixcr#1734] [Q&A]
Hi! 

I am using the Han et all preset (han-et-al-2014) , with mixcr-4.3. I am facing several issues:

1. Low coverage of the plates (I have 20 plates in total, 5 sorted cells in each well, library prep is done the same way as discussed in the paper, and currently libraries are resequenced). However, I am getting quite a similar outcome with the specified preset (~50%). What could be the reason behind it, if not the library prep? 
2.  Some Plate barcodes could not be found in the processed data (plate is missing), however are present in the unprocessed data (with the required seq, as well in the ROW and Column). Is it possible to check the builtin:han-2014-cell3-plate? Or what can be also done in that case? 

Thank you!

**Comment by mizraelson** [comment]
Hi,
1) The preset is optimized to work with datasets with one cell per well as was done in the original publication.
2) Can you share the report files? 
3) I am attaching the list of the plate barcodes build into the preset.
[han-2014-cell3-plate.txt](https://github.com/user-attachments/files/16482563/han-2014-cell3-plate.txt)

**Comment by alexell22** [comment]
Sure!
[result.align.report.txt](https://github.com/user-attachments/files/16482580/result.align.report.txt)
[result.assemble.report.txt](https://github.com/user-attachments/files/16482581/result.assemble.report.txt)
[result.qc.txt](https://github.com/user-attachments/files/16482582/result.qc.txt)
[result.refine.report.txt](https://github.com/user-attachments/files/16482583/result.refine.report.txt)

Thank you for the list!

**Comment by mizraelson** [comment]
Did you use the same primers as in the paper?

**Comment by alexell22** [comment]
Yes, the same (1st, 2nd reaction and barcoding). We did not include the phenotyping ones, just for TCRs.

**Comment by mizraelson** [comment]
From the reports, I see you are using MiXCR v4.6 not 4.3

Here is the pattern from the preset:
```
 ^NN(CELL3PLATE:N{5})ga(CELL1ROW:N{5})ccagggttttcccagtcacgac(R1:*) \
 ^NN(CELL2COLUMN:N{5})gtcactggatttagagtctctcag(R2:*) |
 ^NN(CELL2COLUMN:N{5})gagatctctgcttctgatggctc(R2:*)
 ```
According to the reports, the main issue is that this pattern wasn't matched:

`Alignment failed: absent barcode: 4197136 (94.79%)`

Do you see these primer sequences in your data? If you can share the FASTQ files, I can take a look.

**Reply by alexell22** [reply]
Hi!
Yes, I see, but you are right - not that many (not that many barcodes for the 20th plate; this is not detected at all).
<img width="693" alt="Screenshot 2024-08-06 at 10 35 09" src="https://github.com/user-attachments/assets/b2f37015-b026-4262-bba2-0f153567d5f2">
<img width="709" alt="Screenshot 2024-08-06 at 10 35 23" src="https://github.com/user-attachments/assets/d75edbd1-c569-469f-9ff8-d46136d05e38">
<img width="777" alt="Screenshot 2024-08-06 at 10 43 21" src="https://github.com/user-attachments/assets/282de2c3-2af9-4e84-9557-b5970a6f90c9">

**Reply by alexell22** [reply]
I can also share the files, but I cannot upload them here (file size problem). What would be the better option for you? Would drive link work?

**Reply by mizraelson** [reply]
The drive link would work. You can send it to support@milaboratories.com and I will take a look.

**Reply by alexell22** [reply]
Thank you, have shared the files! :)

**Reply by mizraelson** [reply]
Hi, is it intentional that you have all reads as Undetermined? I checked the FASTQ file and most reads (>70%) align to PhiX which is expected for the Undetermined FASTQ file. The rest does not really seem to have the primers sequences in it even if it aligns to TCR.

For example:


```
@M01338:7:000000000-LL5J2:1:1101:15600:2226 2:N:0:0
TCGCAACACCTTTTTCAGGTCCTCTGTCACCGTGAGCCGTGTGCCCGGCCCGAAGTACTGCTCGTTCGCCCGCCCCTCCTCAAGCCAACCAATTCAGCTTTGGAACCACCTCTCTCCAGTCTCTTCAATACTTCCTGTCAGACTGGACAAGGTAGGGCTTGCATGCTTGGTGGGCAACTTGTCCTTCTCGACACCTTGCTCGTATGTGGCCTTGGAGCCCTCATTGGAAGTTGCCATCAGCATGAGACTCTGTTTCGGGAACTGAAGATACCAAAACATAGTTTTAGCCTGAAATTACAGA
+
-AC-,@FGG,FG,EFGGGGGGGGGG86,CF,BFGGGGG78CG,EF,CGGGGGGGG6CFGCGGG,+,,,+8C+++++,,,,,,,,:F,E,B4,,,C,C?,,C,,,C,A,+4,,CF,,A,<,,C,,,,,45,,,,8,8,4,,,4,4@,,6@E,,3,,+>@E8+363C,@,=+@F+++,3@CCCEG7CF,,7*;9**4;7=D+;?;DF49+/CFFGA4*)22:66)+=64*80AFEF*8<A=*4+4*/4433A@EEEFF:0(1:60)-))..-60348:?<<42::)-)(-5><0,2)6))64)
```

The read aligns to TCR:

<img width="1018" alt="image" src="https://github.com/user-attachments/assets/0680335e-d1d0-4282-a2c7-3882792eea13">

But, the structure MiXCR is looking for in R2 is:

^NN(CELL2COLUMN:N{5})gtcactggatttagagtctctcag(R2:*) or ^NN(CELL2COLUMN:N{5})gagatctctgcttctgatggctc(R2:*)
 
 Thus it is not clear where is the expected CELL2COLUMN barcode.

**Reply by alexell22** [reply]
Hello! Thank you for the response! This is something that we got from the core facility after sequencing. Okay, I see, is it possible to drop the CELL2COLUMN in the preset? Would it be also possible to have an online meeting with you for the discussion?

**Reply by mizraelson** [reply]
Of course, I will forward the meeting details via email.
